src/local_neighborhood.py

```diff
@@ -0,0 +1,121 @@
+        else:
+            raise ValueError("LocalNeighborhood requires nodes and networks")
+
+        #Omics Integrator already gives warnings for strange prize values, so we won't here
+        node_df.to_csv(filename_map['prizes'],sep='\t',index=False,columns=['NODEID','prize'],header=['name','prize'])
+
+        #For now we assume all input networks are undirected until we expand how edge tables work
+        edges_df = data.get_interactome()
+        edges_df['directionality'] = 'U'
+        edges_df.to_csv(filename_map['edges'],sep='\t',index=False,columns=['Interactor1','Interactor2','Weight','directionality'],header=['protein1','protein2','weight','directionality'])
+
+
+    # TODO: implement run function
+    @staticmethod
+    def run(nodetypes=None, network=None, output_file=None, k=None, singularity=False):
+        """
+        Run PathLinker with Docker
+        @param nodetypes:  input node types with sources and targets (required)
+        @param network:  input network file (required)
+        @param output_file: path to the output pathway file (required)
+        @param k: path length (optional)
+        @param singularity: if True, run using the Singularity container instead of the Docker container
+        """
+        # Add additional parameter validation
+        # Do not require k
+        # Use the PathLinker default
+        # Could consider setting the default here instead
+        if not nodetypes or not network or not output_file:
+            raise ValueError('Required PathLinker arguments are missing')
+
+        work_dir = '/spras'
+
+        # Each volume is a tuple (src, dest)
+        volumes = list()
+
+        bind_path, node_file = prepare_volume(nodetypes, work_dir)
+        print("bind_path: ", bind_path)
+        print("node_file: ", node_file)
+
+        volumes.append(bind_path)
+        print("volumes1: ", volumes)
+
+        bind_path, network_file = prepare_volume(network, work_dir)
+        volumes.append(bind_path)
+        print("volumes2: ", volumes)
+
+        # PathLinker does not provide an argument to set the output directory
+        # Use its --output argument to set the output file prefix to specify an absolute path and prefix
+        out_dir = Path(output_file).parent
+        print("out_dir: ", out_dir)
+        # PathLinker requires that the output directory exist
+        out_dir.mkdir(parents=True, exist_ok=True)
+        bind_path, mapped_out_dir = prepare_volume(str(out_dir), work_dir)
+        volumes.append(bind_path)
+        print("bind_path: ", bind_path)
+        print("mapped_out_dir: ", mapped_out_dir)
+        print("bind_path: ", bind_path)
+        mapped_out_prefix = mapped_out_dir + '/out'  # Use posix path inside the container
+
+        command = ['python',
+                   '/PathLinker/run.py',
+                   network_file,
+                   node_file,
+                   '--output', mapped_out_prefix]
```

Comment:
Update the command to match how Local Neighborhood was run outside of the Docker container